docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/biography/node--cgov-biography--full.html.twig

```diff
@@ -1,111 +1,124 @@
+                {% endif %}
+              </p>
               {% endif %}
-              {% if content.field_org_name_2 %}
-              {{ content.field_org_name_2 }}
+                
+
+              {% if content.field_campus|field_value or content.field_room_number|field_value %}
+              <p>
+                {% if content.field_campus|field_value %}
+                  {{ content.field_campus }}
+                {% endif %}
+
+                {% if content.field_campus|field_value and content.field_room_number|field_value %}
+                  |
+                {% endif %}
+
+                {% if content.field_room_number|field_value %}
+                  {{ content.field_room_number }}
+                {% endif %}
+              </p>
               {% endif %}
-              {% if content.field_org_url_2 %}
-              </a>
+
+
+              {% if content.field_phone_number|field_value or content.field_email_address|field_value %}
+              <div class="profile-contact-info">
+                {% if content.field_phone_number|field_value %}
+                  <div class="profile-contact-phone">{{ content.field_phone_number|field_value }}</div>
+                {% endif %}
+                {% if content.field_phone_number|field_value and content.field_email_address|field_value %}
+                  <span class="contact-divider">|</span>
+                {% endif %}
+                {% if content.field_email_address|field_value %}
+                  <div class="profile-contact-email">
+                    <a href="mailto:{{ content.field_email_address|field_value }}">
+                      {{ content.field_email_address|field_value }}
+                    </a>
+                  </div>
+                {% endif %}
+              </div>
               {% endif %}
-            </p>
-            {% if content.field_campus or content.field_room_number %}
-            <p>
-            {% endif %}
-              {% if content.field_campus %}
-              {{ content.field_campus }}
+
+              {% if content.field_twitter_handle or content.field_linkedin_handle %}
+                <div class="profile-panel-social-media">
+                  {% if content.field_twitter_handle %}
+                    <a href="http://www.twitter.com/{{- content.field_twitter_handle|field_value -}}" class="profile-social-link">
+                      <i class="icon twitter" aria-hidden="true"></i>
+                      <span aria-hidden="true">@tweetme</span>
+                      <span class="show-for-sr">Follow me on Twitter</span>
+                    </a>
+                  {% endif %}
+                  {% if content.field_linkedin_handle|field_value %}
+                    <a href="http://www.linkedin.com/in/{{- content.field_linkedin_handle|field_value -}}" class="profile-social-link ">
+                      <i class="icon linkedin" aria-hidden="true"></i>
+                      <span aria-hidden="true">/linkme</span>
```

Comment:
Victoria just showed me this: as per requirements on https://github.com/NCIOCPL/cgov-digital-platform/issues/664, there should be no "/linkme" text displaying.